src/components/slider/Slider.module.css

```
@@ -0,0 +1,262 @@
+  position: absolute;
+  top: 100%;
+  left: 50%;
+  transform: translateX(-50%);
+  width: 0;
+  height: 0;
+  border-left: 6px solid transparent;
+  border-right: 6px solid transparent;
+  border-top: 6px solid var(--surface-secondary);
+}
+
+@keyframes tooltipFadeIn {
```

Stylelint (17.3.0)

[error] Expected keyframe name "tooltipFadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)